src/widgets/AmountPerUnitWidget.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace hipanel\modules\finance\widgets;
+
+use Yii;
+use yii\base\Widget;
+use yii\bootstrap\Html;
+
+final class AmountPerUnitWidget extends Widget
```

Comment:
```suggestion
final class PricePerUnitWidget extends Widget
```

Amount – это как раз Price*Quantity
А per unit – это Price